src/schoology-extractor/schoology_extractor/mapping/discussion_replies.py

```diff
@@ -0,0 +1,87 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+from datetime import datetime
+import pandas as pd
+
+from . import constants
+
+DISCUSSION_REPLIES_TYPE = "Discussion Reply"
+
+
+def map_to_udm(discussion_replies_df: pd.DataFrame, section_id: int) -> pd.DataFrame:
+    """
+    Maps a DataFrame containing Schoology section associations (enrollments)
+    into the Ed-Fi LMS Unified Data Model (UDM) format.
+
+    Parameters
+    ----------
+    enrollments_df: DataFrame
+        Pandas DataFrame containing Schoology assignments for a section
+
+    Returns
+    -------
+    DataFrame
+        A LMSUsers-formatted DataFrame
+
+    Notes
+    -----
+    DataFrame columns are:
+        SourceSystemIdentifier: A unique number or alphanumeric code assigned to a the discussion-reply by
+            the source system
+        SourceSystem: The system code or name providing the user data
+        LMSUserSourceSystemIdentifier: A unique number or alphanumeric code assigned to a user by the source
+            system
+        LMSSectionSourceSystemIdentifier: A unique number or alphanumeric code assigned to a section by the
+            source system
+        EntityStatus: The status of the record
+        ActivityDateTime - The date/time the replied was created.
```

Comment:
Inconsistent separators here and next several lines, e.g.
```suggestion
        ActivityDateTime: The date/time the replied was created.
```

Reply:
✔️